app/learning/index.md

```diff
@@ -5,204 +5,98 @@ title: Getting started with Yeoman
 sidebar: sidebars/learning.md
 ---
 
-The Yeoman workflow is comprised of three core tools for improving your productivity and satisfaction when building a web app. These tools are:
+Yeoman is a generic scaffolding system allowing you to create any kind of app. It'll allow you to rapidly get started on new projects and streamline the maintenance of existing projects.
 
-* [yo](https://github.com/yeoman/yo) - the scaffolding tool from Yeoman
-* A package manager, such as [bower](http://bower.io/) or [npm](https://www.npmjs.com/).
-* A build tool, such as [Grunt](http://gruntjs.com/) or [Gulp](http://gulpjs.com/).
+Yeoman is language agnostic. It can generates projects in any languages (Web, Java, Python, C#, etc.)
 
-Each of these projects are independently maintained by their respective communities, but work well together as a part of a prescriptive workflow for keeping you effective. Let’s walk through what these binaries can do.
+Yeoman by itself doesn't make any decision. Yeoman runs _generators_ who're making every decision. A generator is basically a plugin in the Yeoman environment. There's more than [1500 publicly available generators](/generators) and you can [create your own](/authoring) if public ones are not matching your team workflow; Yeoman is always the right choice for your scaffolding needs.
 
-<p class="toolset">
-  <img class="full" src="/assets/img/workflow.jpg">
-</p>
+Here's a couple common use cases:
 
-## yo
+- Creating new projects rapidly
+- Creating section of your project (how about creating a new controller with unit tests?)
+- Creating packages
+- Bootstrapping new services
+- Enforcing standards and latest best practices in any new app components.
+- Promote your project by allowing users to bootstrap new plugins
+- Etc, etc
 
-Yo is maintained by the Yeoman project and offers web application scaffolding, utilizing scaffolding templates we refer to as generators. You typically install yo and any generators you think you might use via npm.
+## Getting started
 
-### Installing yo and some generators
-
-First, you'll need to install `yo` and other required tools:
-
-```sh
-npm install -g yo bower grunt-cli gulp
-```
+Yo is the Yeoman command line utility allowing you to create projects utilizing scaffolding templates we refer to as generators. You typically install yo and any generators you think you might use via npm.
```

Comment:
`Yo` is the Yeoman command line utility allowing the creation of projects utilizing scaffolding templates (referred to as generators). Yo and the generators used are installed using [npm](http://npmjs.org).